about.html

```diff
@@ -0,0 +1,176 @@
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, height=device-height, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/featuredCourse.css" />
+    <link rel="stylesheet" href="css/desktop.css" />
+    <link rel="stylesheet" href="css/animations.css" />
+    <title>The Code Chaser</title>
+  </head>
+  <body>
+    <!-- #### header-section #### -->
+    <header>
+        <div class="header">
+          <a class="header-menu" href="#"><i class="fab fa-facebook-f"></i></a>
+          <a class="header-menu" href="#"><i class="fab fa-twitter"></i></a>
+          <a class="header-menu" href="#">English</a>
+          <a class="header-menu" href="#">My Page</a>
+          <a class="header-menu" href="#">Logout</a>
+        </div>
+        <div class="header2">
+      <a href="index.html"><img src="images/header-logo.png" alt="header logo" class="header-logo" /></a>
+
+      <!-- #### navbar-section #### -->
+      <nav>
+        <div class="desktop-nav">
+          <ul class="desktop-menu-items">
+            <li>
+            <a class="desktop-menu-item" href="index.html">Home</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#">Program</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#">Tutors</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#partners">Sponser</a>
+             </li>
+             <li>
+            <a class="desktop-menu-item" href="#contact">Contact</a>
+            </li>
+            <li>
+            <a class="desktop-menu-item1" href="#">CC Campaign</a>
+            </li>
+          </ul>
+        </div>
+        <div class="mobile-nav">
+     <button type="button" class="btn-open" ><img alt="hamburger icon" src="images/hamburger-icon.png" class="hamburger-icon" /></button>  
+        <div class="mobile-menu-container">
+              <button type="button" class="btn-close">X</button>
+          <ul class="mobile-menu-items">
+            <li>
+            <a class="mobile-menu-item" href="index.html">Home</a>
+            </li>
+             <li>
+            <a class="mobile-menu-item" href="#">Program</a>
+            </li>
+             <li>
+            <a class="mobile-menu-item" href="#">Tutors</a>
+            </li>
+             <li>
+            <a class="mobile-menu-item" href="#partners">Sponser</a>
+            </li>
+            <li>
+            <a class="mobile-menu-item" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+        </div>
+      </nav>
+      </div>
+    </header>
```

Comment:
- [ ] The hamburger menu should be on the left side, please move it so it matches the design.
- [ ] The `header` is not identical in both pages, kindly update it.

Reference Design
![image](https://user-images.githubusercontent.com/33767951/142396911-b3a97b44-cd7c-48e3-a23d-e49d0ca5d782.png)

Your Styling => `320px`
![image](https://user-images.githubusercontent.com/33767951/142397062-cf2e6238-27ed-4e7f-af8e-cca68eda9421.png)
![image](https://user-images.githubusercontent.com/33767951/142397147-ca89ef86-4cef-421e-ace7-e38f0f7ee100.png)